security/cmd.py

```diff
@@ -38,9 +40,29 @@ def checker():
         if not filename:
             sys.stderr.write("Error: no config file provided\n")
             parser.print_usage()
-    _checker = security.checker.Checker()
+            sys.exit(1)
+    _checker = security_checker.Checker()
     logging.debug("Loading config")
     config = yaml.safe_load(open(filename))
     _checker.configure(config)
     logging.info("Entering loop")
     _checker.run()
+
+
+def api():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config", type=str, help=("config file. defaults "
+                                                    "to $SECURITY_CONF"))
+    args = parser.parse_args()
+    config = args.config
+    if config is None:
+        config = os.environ.get("SECURITY_CONF")
+        if config is None:
+            sys.stderr.write("Error: no config file provided\n")
+            parser.print_usage()
+            sys.exit(1)
+    config = yaml.safe_load(open(config))
+    app = flask.Flask("security", static_folder=None)
```

Comment:
Could you please move the `app` in a separate module because it is not so trivial to run this application in a production mode using `gunicorn`. FYI, `gunicorn` [uses the importable path to a variable with a WSGI application](http://docs.gunicorn.org/en/latest/run.html#gunicorn).